palette/src/alpha/mod.rs

```diff
@@ -0,0 +1,163 @@
+use crate::Component;
+
+#[doc(hidden)]
+pub use palette_derive::Transparency;
+
+pub use self::alpha::*;
+
+mod alpha;
+
+/// A trait for color types that can have or be given transparency (alpha channel).
+///
+/// It's an interface for adding, removing and setting the transparency of a
+/// color type. The type itself doesn't need to store the transparency
+/// value, as long as it can be transformed into or wrapped in a type that
+/// does have a representation of transparency. This would typically be done
+/// by wrapping it in an [`Alpha`](struct.Alpha.html) instance.
```

Comment:
```rust
/// `Transparency` is an interface for adding, removing and setting the alpha
/// channel of a color type. The type itself doesn't need to store the
/// transparency value as it can be transformed into or wrapped in a type that
/// has a representation of transparency. This would typically be done by
/// wrapping it in an [`Alpha`](struct.Alpha.html) instance.
```